BootloaderCorePkg/Tools/CommonUtility.py

```diff
@@ -206,6 +206,10 @@ def get_openssl_path ():
 
 def run_process (arg_list, print_cmd = False, capture_out = False):
     sys.stdout.flush()
+
+    if os.name == 'nt' and '.exe' not in arg_list[0]:
+        arg_list[0] += '.exe'

```

Comment:
Better check if there is file extension exist instead of checking '.exe' in arg_list.
For example, if a test.bat given, it should not append .exe.

Reply:
There might be more cases that need to be handled.  For example,  if the executable is "test.cmd",  then this logic will break it.  Similar for other Windows script, like vbs, ps, etc.

Reply:
How about change to the line below?
if os.name == 'nt' and os.path.splitext(arg_list[0])[1] == '':

Reply:
Yes, that should work better.